src/Sylius/Bundle/ShopBundle/Twig/Component/Cart/WidgetComponent.php

```diff
@@ -46,23 +50,26 @@ public function initializeCart(): void
 
     #[LiveListener(FormComponent::SYLIUS_SHOP_CART_CHANGED)]
     #[LiveListener(FormComponent::SYLIUS_SHOP_CART_CLEARED)]
-    public function refreshCart(#[LiveArg] ?Order $cart = null): void
+    public function refreshCart(#[LiveArg] ?int $cart = null): void
```

Comment:
As the id may not be an integer:
```suggestion
    public function refreshCart(#[LiveArg] mixed $cartId = null): void
```